busy_beaver/tasks/toolbox.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+from flask import current_app as app  # noqa
+from rq import get_current_job
+
+from busy_beaver.extensions import db
+from busy_beaver.models import Task
+
+logger = logging.getLogger(__name__)
+
+
+def _set_task_progress(progress):
+    job = get_current_job()
```

Comment:
I envision building out a task endpoint that can be queried to get the status and/or result of a task